hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -188,11 +188,11 @@ public class HoodieCompactionConfig extends HoodieConfig {
       .withDocumentation("Log compaction can be scheduled if the no. of log blocks crosses this threshold value. "
           + "This is effective only when log compaction is enabled via " + INLINE_LOG_COMPACT.key());
 
-  public static final ConfigProperty<String> USE_LOG_RECORD_READER_SCAN_V2 = ConfigProperty
-      .key("hoodie.log.record.reader.use.scanV2")
+  public static final ConfigProperty<String> ENABLE_OPTIMIZED_LOG_BLOCKS_SCAN = ConfigProperty
+      .key("hoodie.optimized.log.blocks.scan.enable")
       .defaultValue("false")
```

Comment:
`enable` or `enabled`? Things kind of become messy because in `HoodieWriteConfig` some option keys are suffixed with `enable` and some are with `enabled`. Somehow, we need to unify the naming pattern.